Makefile.validation

```diff
@@ -143,7 +143,37 @@ kind-install: kind-build
 	kind version
 
 kind-create-cluster:
-	$(KIND) create cluster --config $(KIND_DIR)/kind.yaml --image kindest/node:v1.18.19
+	$(KIND) create cluster --config $(KIND_DIR)/kind.yaml --image kindest/node:v1.18.2
 
 kind-delete-cluster:
 	$(KIND) delete cluster
+
+sidecars-pull:
```

Comment:
Please remove these changes

Reply:
Removed. Sorry, I'm confused in my branches